src/actions/modal.js

```diff
@@ -0,0 +1,37 @@
+import {
+  MODAL_OPEN,
```

Comment:
If we have MODAL_OPEN, we should use it as toggle param. I think that we don't need the MODAL_CLOSE. Let's name it as MODAL_TOGGLE and the payload will be a bool param